CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+[1.3.0] - 2021-03-24
```

Comment:
You will need to update the date here when the actual release is published. @ffmancera when is this planned?

Reply:
In the moment it is approved. Maybe we can say today? :-)

Reply:
I updated into `[1.3.0] - 2021-04-08`

Reply:
I updated into `[1.3.0] - 2021-04-08`